src/components/CreateSphere.jsx

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable react/no-access-state-in-setstate */
+/* eslint-disable react/destructuring-assignment */
+import React, { Component } from "react";
+import styled from "styled-components";
```

Comment:
This is what is causing the build to fail, `styled-components` is not installed (though it is installed in the most recent commit on main branch, so just merge those changes into this branch)